template-parts/content-image-card.php

```diff
@@ -35,10 +35,8 @@
 <li class="mdc-image-list__item">
 	<a href="<?php the_permalink(); ?>">
 		<img class="mdc-image-list__image" src="<?php echo esc_url( $thumbnail ); ?>">
```

Comment:
@ravichdev Removing the extra link and having both image and text in the same link ensures the link text is read instead of an image. `alt` needs to describe the image so we can't just add a generic one, but I can add it so it reads what the media library has

Reply:
That makes sense, we can keep it this way then and not add an alt text.

Reply:
Confirmed in the theme review slack alt text should be empty:

> For a featured image inside a link, what matters is that the link has to have text. If the link wraps both a featured image and a post title as a block-level link, then the alt attribute should be empty. If the link only wraps the image, then the alt text needs to describe the link rather than the image - so it should be the post title.